Analysis/Tasks/trackqa.cxx

```diff
@@ -144,6 +143,41 @@ struct TrackQATask {
   }
 };
 
+struct TrackCutQATask {
+  OutputObj<TH1F> passed_tracks{TH1F("hTrackCut", "hTrackCut", TrackSelection::kNCuts, 0, TrackSelection::kNCuts), OutputObjHandlingPolicy::QAObject};
+  TrackSelection selectedTracks;
+
+  void init(InitContext&)
+  {
+    selectedTracks.SetTrackType(aod::track::Run2GlobalTrack);
+    selectedTracks.SetPtRange(0.1f, 1e10f);
+    selectedTracks.SetEtaRange(-0.8f, 0.8f);
+    selectedTracks.SetRequireITSRefit(true);
+    selectedTracks.SetRequireTPCRefit(true);
+    selectedTracks.SetRequireGoldenChi2(true);
+    selectedTracks.SetMinNCrossedRowsTPC(70);
+    selectedTracks.SetMinNCrossedRowsOverFindableClustersTPC(0.8f);
+    selectedTracks.SetMaxChi2PerClusterTPC(4.f);
+    selectedTracks.SetRequireHitsInITSLayers(1, {0, 1}); // one hit in any SPD layer
+    selectedTracks.SetMaxChi2PerClusterITS(36.f);
+    selectedTracks.SetMaxDcaXYPtDep([](float pt) { return 0.0105f + 0.0350f / pow(pt, 1.1f); });
+    selectedTracks.SetMaxDcaZ(2.f);
```

Comment:
Since this is the same as getGlobalTrackSelection() it might be good to just use that function, right?
We could move the default Cut-Settings that are currently in the trackselection.cxx task to a separate file e.g. TrackSelectionDefaults.h. This would allow to re-use them in different tasks.

Reply:
Yes sure, as in the current aliphysics!